plugins/modules/wafv2_web_acl.py

```diff
@@ -105,6 +101,7 @@
 extends_documentation_fragment:
 - amazon.aws.aws
 - amazon.aws.ec2
+- amazon.aws.tags
 
```

Comment:
Do we need to document when `purge_tags` has been added with a note?

Reply:
Yup, we probably should